authentication/managing_cloud_provider_credentials/cco-mode-sts.adoc

```diff
@@ -0,0 +1,734 @@
+[id="cco-mode-sts"]
+= Using manual mode with STS
+include::modules/common-attributes.adoc[]
+:context: cco-mode-sts
+toc::[]
+
+Manual mode with STS is available as a link:https://access.redhat.com/support/offerings/techpreview[Technology Preview] for Amazon Web Services (AWS).
```

Comment:
It seems you modeled the existing topics. Not sure we should lead with this paragraph, rather describe what each mode is then state that it's TP. If I don't know what the mode is, why do I care if it's supported? Just a thought.

Reply:
I'm worried about not having the supported platforms up front on these, because I would hate for someone using e.g. RHV to have to read all about mint mode and then find out they can't use it. This particular page is I guess a bit odd in that we can't just say it's supported on AWS, since it's not supported. But that would change to the more normal statement when this is GA.